pkg/catalog/diagram.go

```diff
@@ -120,7 +169,7 @@ func (p *Generator) CreateSequenceDiagram(appName string, endpoint *sysl.Endpoin
 		os.Exit(1)
```

Comment:
unrelated but idk why we're os.exiting here

Reply:
> Is there a way for github actions bot sending the notification only once?

I might look into that next, there doesn't seem to be much de-duplication built into Github Actions for posting comments from what I could find 👍